dragonchain/webserver/lib/blocks.py

```diff
@@ -59,6 +60,10 @@ def query_blocks_v1(params: Optional[dict], parse: bool) -> "RSearch":
         return redisearch.search(index="bk", query_str="*", sort_by="block_id", sort_asc=False)
 
 
+def list_blocks_v1(get_whole_object: bool) -> List[str]:
+    return storage.list_objects("BLOCK")
```

Comment:
We definitely can't do this, there are verifications blocks in there too for L1's. They'll need to be filtered out